From 48026e164da5704411c61db9df304f314c97a9fd Mon Sep 17 00:00:00 2001
From: Sudarshan Rajagopalan <sudaraja@codeaurora.org>
Date: Tue, 6 Oct 2020 11:31:04 -0700
Subject: [PATCH 790/830] ANDROID: gki_defconfig: Enable
 CONFIG_MEMORY_HOTREMOVE
X-NVConfidentiality: public

With the support for memory hotplug and hotremove fully
functional, and having configs ARCH_ENABLE_MEMORY_HOTPLUG
and ARCH_ENABLE_MEMORY_HOTREMOVE enabled by default for
arm64 targets, enable config MEMORY_HOTREMOVE.

Bug: 170202777
Change-Id: I618dfeeb3f0f4f2c347e6f6424303725bf15e461
Signed-off-by: Sudarshan Rajagopalan <sudaraja@codeaurora.org>
---
 arch/arm64/configs/gki_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 445bd0644695..2051760785d9 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -93,6 +93,7 @@ CONFIG_GKI_HACKS_TO_FIX=y
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
 CONFIG_BINFMT_MISC=y
 CONFIG_MEMORY_HOTPLUG=y
+CONFIG_MEMORY_HOTREMOVE=y
 CONFIG_CLEANCACHE=y
 CONFIG_CMA=y
 CONFIG_CMA_AREAS=16
-- 
2.17.1

